idea/src/META-INF/plugin.xml

```diff
@@ -1520,6 +1520,15 @@
                      language="kotlin"
     />
 
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.inspections.UnnecessaryJavaUsageInspection"
+                     displayName="Unnecessary java usage"
```

Comment:
This is no more clear than the same text in the inspection description, and also needs to be changed.